packages/core/tracing/src/utils/lifecycle.ts

```diff
@@ -145,7 +149,7 @@ export const buildLifecycleGraph = (root: SpanNode): LifecycleGraph => {
     nodeTree: {
       client: {
         node: {
-          id: LifecycleNodeType.CLIENT,
+          id: `${options?.idPrefix ?? ''}${LifecycleNodeType.CLIENT}`,
```

Comment:
@sumimakito WDYT about evaluate the expression only once on the top of the `buildLifecycleGraph` method 🤔 

```ts
export const buildLifecycleGraph = (root: SpanNode, options?: BuildLifecycleGraphOptions): LifecycleGraph => {
  const prefix = options?.idPrefix ?? '';
  // while using
{ 
  // ...other attributes
  id: `${prefix}${LifecycleNodeType.CLIENT_OUT}`,
}
```

Reply:
Good idea and I've just pushed another commit